docs/getting-started-on-snowplow-bdp-enterprise/setup-guide-azure/index.md

```diff
@@ -1,8 +1,86 @@
 ---
 title: "Setup Snowplow BDP on Azure"
+date: "2024-01-11"
 sidebar_position: 30
 ---
 
-## Coming soon
+## Request your pipeline through Snowplow BDP Console
 
-If you are interested in deploying [BDP Enterprise](https://snowplow.io/pricing/) in your Azure account, please [join our waiting list](https://snowplow.io/bdp-enterprise-on-azure/).
+Login to Snowplow BDP Console where you'll be able to follow a step-by-step guide to getting started (including the steps below).
+
+## Setting up your Azure account
+
+To get your cloud environment ready for your Snowplow pipeline to be installed:
+
+### Create a new account
+
+Please create a new Azure account and tenant segregated from anything else you have running in Azure. For the latest documentation on setting up a new account and tenant please refer to the Azure documentation [here](https://azure.microsoft.com/).
+
+
+### Enable billing for the account
+
+Please enable billing in the tenant by creating a subscription as failing to do so will prevent the pipeline from being installed. For details on enabling billing for your tenant, please refer to the Azure documentation for [subscriptions](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription).
+
+
+### Set up Access and Required Permissions
+
+Snowplow deploys into your tenant using a verified [application service principle](https://learn.microsoft.com/en-us/entra/identity-platform/app-objects-and-service-principals?tabs=browser#service-principal-object) (Enterprise application). We require a custom role to be assigned to the application service principle to allow creation of custom pipeline roles needed for deploying and managing different components of your infrastructure.
```

Comment:
```suggestion
Snowplow deploys into your tenant using a verified [application service principal](https://learn.microsoft.com/en-us/entra/identity-platform/app-objects-and-service-principals?tabs=browser#service-principal-object) (Enterprise application). We require a custom role to be assigned to the application service principal. This will allow us to create custom pipeline roles needed for deploying and managing different components of your infrastructure.
```

Reply:
Oh! Good catch here!